changelogs/fragments/unittest_fix_for_resource_module_base.yaml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
```

Comment:
We have marked this as `trivial` on other platforms.

Reply:
updated